Interop/StubGenerator/src/test/kotlin/org/jetbrains/kotlin/native/interop/gen/J2ObjcManglingTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.kotlin.native.interop.gen
```

Comment:
This test is good it's likely worth while seeing if there is anything we should add to the system test.  (especially if we can hook up actual j2objc generated code now.

This would better validate that everything is working as expected.

Reply:
Added to my list, I will have to change the tests now already because of this changing the method names.

Reply:
That should likely be part of this PR right?